tests/__mocks__/vscode.ts

```diff
@@ -22,7 +22,7 @@ const window = {
 };
 
 const workspace = {
-   rootPath: "master"
+   rootPath: "develop"
```

Comment:
Why using `develop` here ? I mean we mock a Git workspace here, and `master` is a default branch always present in Git.

Reply:
I've changed it to not override this variable in specs, but I will swap it :)